tests/base.py

```diff
@@ -1,11 +1,12 @@
 # -*- encoding:utf-8 -*-
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-import unittest
+import pytest
 
 import patchy.api
 
 
-class PatchyTestCase(unittest.TestCase):
-    def setUp(self):
+class PatchyTestCase(object):
```

Comment:
can you put this unittest -> pytest stuff in a separate PR please, it's adding noise to this commit